src/routes/io/io.py

```diff
@@ -80,3 +79,45 @@ async def analyze_img_file(
             "deduced-mime-type": deduced_type})
     except:
         raise internal_error_500()
+
+file_data = {}
+  
+@io_router.post('/user')
+async def create_user(username:str
+,    email:str
+,    password:str
+):
+    try:
+        file_data[username] = {
+            "username" : username,
+            "email" : email,
+            "password": password
+        }
+        return ok_200({"username":username,
+                       "email" : email,
+                       "password": password})
+    except:
+        raise internal_error_500()
+
+@io_router.put('/user')
+async def create_user(old_username:str
```

Comment:
Also username is unique, it should not change. There is no use of old_username and new_username.

Reply:
ok I will give functionality only to email and password